.github/workflows/ci.yaml

```
@@ -222,3 +222,11 @@ jobs:
           # See the License for the specific language governing permissions and
           # limitations under the License.
           "
+
+  verify-no-conflicts:
+    name: Verify no conflicts
```

Minor naming suggestion
```suggestion
  verify-compatible-requirements:
    name: Verify compatible requirements
```